File: src/cn/langya/sun/value/BoolValue.java
package cn.langya.sun.value;

public class BoolValue extends ValueManager {
    private boolean state = false;
    private String name = null;

    public BoolValue(String name, boolean state) {
        this.name = name;
        this.state = state;
    }

    public void set(boolean value) {
        this.state = value;
    }

    public boolean get() {
        return this.state;
    }
}

File: src/cn/langya/sun/value/BoolValue.kt
package cn.langya.sun.value


/**
 * @author LangYa
 * @ClassName BoolValue
 * @date 2023/12/15 17:19
 * @Version 1.0
 */

class BoolValue(name: String,state: Boolean): ValueManager() {

    var state: Boolean = false

    fun set(boolean: Boolean) {
        state = boolean
    }

    fun get(): Boolean {
        return state
    }

}